src/Microsoft.VisualStudio.Services.Agent/AgentService.cs

```diff
@@ -19,8 +19,8 @@ public interface IAgentService
 
     public abstract class AgentService
     {
-        protected IHostContext HostContext { get; private set; }
```

Comment:
I had to resolve circular dependency and in SecretMasker.Initialize i could not call base.Initialize